test/controllers/ActiveBankAccountControllerSpec.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.ActiveBankAccountController._
+import forms.YesNoPageFormProvider
+import models.NormalMode
+import views.html.YesNoPageView
+
+class ActiveBankAccountControllerSpec extends ControllerBaseSpec {
+
+  private lazy val onPageLoad = routes.ActiveBankAccountController.onPageLoad(srn, NormalMode)
+  private lazy val onSubmit = routes.ActiveBankAccountController.onSubmit(srn, NormalMode)
+
+  "ActiveBankAccountController" - {
+
+    act.like(renderView(onPageLoad) { implicit app => implicit request =>
```

Comment:
Need a test for rendering a page that is prepopulated
```
    act.like(renderPrePopView(onPageLoad, PersonalContributionsPage(srn), true) { implicit app => implicit request =>
      injected[YesNoPageView].apply(form(injected[YesNoPageFormProvider]).fill(true), viewModel(srn, NormalMode))
    })
```

Need two tests for it redirecting to the next page as well
```
    act.like(redirectNextPage(onSubmit, "value" -> "true"))
    act.like(redirectNextPage(onSubmit, "value" -> "false"))
```